From 253e3c5d4f2cd96def31b3416d0d6ca42a8bb271 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Tue, 19 Nov 2013 16:32:52 +0800
Subject: [PATCH 0745/1072] ENGR00288578-5 usb: chipidea: change some output
 message

git://git.freescale.com/imx/linux-2.6-imx.git imx_3.10.17_1.0.0_beta
commit 8952f0f3e3de66ea7997cfd3a2cf409765250e4c

To reduce the message output when unload the module

Signed-off-by: Peter Chen <peter.chen@freescale.com>
(cherry picked from commit 910aebacf698995408827f40ca269650afc384e0)
Signed-off-by: Hongbo Zhong <hongbo.zhong@windriver.com>
---
 drivers/usb/chipidea/otg.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/chipidea/otg.c b/drivers/usb/chipidea/otg.c
index bdf3f41..075d0b9 100644
--- a/drivers/usb/chipidea/otg.c
+++ b/drivers/usb/chipidea/otg.c
@@ -113,7 +113,7 @@ static void ci_otg_event(struct ci_hdrc *ci)
 		ci_handle_vbus_change(ci);
 		pm_runtime_put_sync(ci->dev);
 	} else
-		dev_err(ci->dev, "unexpected event occurs at %s\n", __func__);
+		dev_dbg(ci->dev, "it should be quit event\n");
 
 	enable_irq(ci->irq);
 }
-- 
1.7.5.4

